Swift/QtUI/QtXMLSenderWidget.h

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
Is an additional widget really needed in this case, or wouldn't it be more straightforward to just put QTextEdit in the existing QtXMLConsoleWidget class.  I think that would make the code a bit cleaner.